docs/user/TerraformAWSAuthentication.md

```diff
@@ -1,8 +1,8 @@
 # Authenticating Terraform to Access AWS
 
-In order to create and destroy AWS resources on your behalf, Terraform needs a AWS account that has sufficient permissions to perform all the actions defined in the Terraform manifest. You will need an AWS account IAM user that has at a minimum the permissions listed in [this policy](../../files/policies/devops-iac-eks-policy.json).
+In order to create and destroy AWS resources on your behalf, Terraform needs to authenticate with credentials associated with an AWS account IAM user or credentials generated by assuming an IAM role that has sufficient permissions to perform all the actions defined in the Terraform manifest. You will need either an AWS account IAM user or an AWS IAM role that has at a minimum the permissions listed in [this policy](../../files/policies/devops-iac-eks-policy.json).
```

Comment:
In order to create and destroy AWS resources on your behalf, Terraform needs credentials that enable it to authenticate to AWS. Two types of credentials are acceptable: 
- Credentials that are associated with an AWS IAM user account, or 
- Credentials that are generated by assuming an IAM role that has sufficient permissions to perform all the actions that are defined in the Terraform manifest. 

You will need either an AWS IAM user account or an AWS IAM role that has at a minimum the permissions listed in [this policy](../../files/policies/devops-iac-eks-policy.json).

Reply:
suggestions applied, the upcoming commit will contain the updates for your review